### PR TITLE
[Footer] Add 'Show policy links' setting

### LINKED ISSUE #2091: 🌐 New policies setting in Footer
User story: As merchant on flagship themes, I can show links to policies in my footer. 

This ticket introduces the new setting to let merchants on flagship themes display their policy links in their footer.

### PR DESCRIPTION
**Pull request by LucasLacerdaUX**
### PR Summary: 
We introduced a new theme setting that allows you to show policy links in the Footer section. Policies can be edited on the  Store Settings.

### Why are these changes introduced?
Fix #2091 

### What approach did you take?
Using `shop.policies` object, we loop through all the policies and check if they're not empty. The `privacy.title` property is then used to render the title.

We could've used the individual `shop.shipping_policy`, `shop.subscription_policy` links and render them manually, but this approach would make the theme less future-proof.

### Testing steps/scenarios

- [ ] Edit your store's Policies in the Store Settings page.
- [ ] Check `Show policy links` setting on the Footer section.
- [ ] Make sure no empty policies are showing.

### Demo links

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=136847261718)
- [Editor](https://os2-demo.myshopify.com/admin/themes/136847261718/editor)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Comment by nicklepine** [comment]
Looks great. 
image

Looking at other help texts in the footer, I think we could respect the same format: 
- [ ] Proposed new copy: "To add store policies, go to your policy settings."
- [ ] "policy settings" should be linking out to /admin/settings/legal

**Comment by allylane** [comment]
Looks great!

**Review by allylane — DISMISSED** [review]

File: assets/section-footer.css
Comment (allylane) [reply assets/section-footer.css:510]: @ludoboludo @LucasLacerdaUX Agree, adding some padding here would be good.

Comment (LucasLacerdaUX) [reply assets/section-footer.css:508]: I've updated to add `1rem` padding at top and bottom. Let me know if this looks good or which value I should tweak to if needed.

File: assets/section-footer.css
Comment (ludoboludo) [reply assets/section-footer.css:510]: I think it looks a bit nicer at the bottom below the rest on a new row. With maybe a bit more white space in between the first and the second row. But when it's short without the policy links it's not bad under the payment icons. 🤔

File: assets/section-footer.css
Comment (ludoboludo) [inline assets/section-footer.css:264]: Wondering if we should keep this or why we need it 🤔 
[Video](https://screenshot.click/22-09-yr607-cnnpt.mp4)

File: assets/section-footer.css
Comment (ludoboludo) [inline assets/section-footer.css:305]: Curious as to why this is needed as well. Seem like that element contains the button and the container showing when the button is clicked but since it's absolutely positioned I'm not too sure what it does 🤔

File: assets/section-footer.css
Comment (ludoboludo) [reply assets/section-footer.css:510]: I think once we settle on a decision here I'm ok to approve 👍

Comment (LucasLacerdaUX) [reply assets/section-footer.css:489]: @ludoboludo @sofiamatulis @allylane We decided to move the policy links and copyright text to it's own line to avoid the issues [mentioned above](https://github.com/Shopify/dawn/pull/2084#discussion_r1019550762).

@allylane suggested above that we should center align this content, but I personally think that left aligned looks better in most scenarios.

### Left align
#### Payment Icons ON. Policy Links OFF.
image

#### Payment Icons ON. Policy Links ON.
image

#### Payment Icons OFF. Policy Links OFF.
image

#### Payment Icons OFF. Policy Links ON.
image

### Center aligned
#### Payment Icons ON. Policy Links OFF.
image

#### Payment Icons ON. Policy Links ON.
image

#### Payment Icons OFF. Policy Links OFF.
image

#### Payment Icons OFF. Policy Links ON.
image

Do we want to move forward with `center` aligned, like originally suggested, or should we consider this `left` aligned?

Comment (LucasLacerdaUX) [reply assets/section-footer.css:489]: @ludoboludo @sofiamatulis @allylane We decided to move the policy links and copyright text to it's own line to avoid the issues [mentioned above](https://github.com/Shopify/dawn/pull/2084#discussion_r1019550762).

@allylane suggested above that we should center align this content, but I personally think that left aligned looks better in most scenarios.

### Left align
#### Payment Icons ON. Policy Links OFF.
image

#### Payment Icons ON. Policy Links ON.
image

#### Payment Icons OFF. Policy Links OFF.
image

#### Payment Icons OFF. Policy Links ON.
image

### Center aligned
#### Payment Icons ON. Policy Links OFF.
image

#### Payment Icons ON. Policy Links ON.
image

#### Payment Icons OFF. Policy Links OFF.
image

#### Payment Icons OFF. Policy Links ON.
image

Do we want to move forward with `center` aligned, like originally suggested, or should we consider this `left` aligned?

**Review by LucasLacerdaUX — COMMENTED** [review]
Updates:
- Policy links and copyright now live on their own line.
- Reverted changes to localization forms. PR should be easier to review now.

File: assets/section-footer.css
Comment (LucasLacerdaUX) [reply assets/section-footer.css:487]: Final decision: Updated to `0.6rem`.

**Comment by allylane** [comment]
I agree that left aligned looks better @LucasLacerdaUX 

The text is a bit small on mobile. Can we increase it? And the variation in widths of inputs + mixed alignment is making the footer look a bit messy. Is that something we should address as a separate PR?

Screenshot 2022-12-08 at 9 35 09 AM

**Comment by LucasLacerdaUX** [comment]
> The text is a bit small on mobile. Can we increase it?
Agreed. I've just updated it to be `1rem` = `10px` so it's consistent with desktop.

> And the variation in widths of inputs + mixed alignment is making the footer look a bit messy. Is that something we should address as a separate PR?

Ideally, yes. We can ship it as part of the same update, but this would keep the scope smaller for this PR and also unblock us to ship any features that are depending on this setting.

File: assets/section-footer.css
Comment (sofiamatulis) [inline assets/section-footer.css:490]: Ive seen this being used in some other places in the theme like blog. I wonder if we can re-use the class? And just update the spacing?
 https://screenshot.click/08-42-9mpqo-wa4u8.png

File: locales/vi.schema.json
Comment (sofiamatulis) [inline locales/vi.schema.json:815]: Do we need the circle if it goes to the 2nd line? (first circle)

https://screenshot.click/08-45-wd8so-jri89.png

cc @allylane

**Review by sofiamatulis — COMMENTED** [review]
Looks good 👍 Added 2 small nitpicks

File: assets/section-footer.css
Comment (LucasLacerdaUX) [reply assets/section-footer.css:490]: That's a different symbol though.

We're using a [`Middle dot` character](https://www.compart.com/en/unicode/U+00B7) while the blog is using a [`Bullet`](https://www.compart.com/en/unicode/U+2022) character. I'm not sure if they're meant to be using the same, though. For policy, we definitely want to keep it as `\00B7` because of it's sizing.

File: assets/section-footer.css
Comment (LucasLacerdaUX) [reply assets/section-footer.css:493]: That's a good point. From an editing perspective, I do worry about introducing some styling magic with things moving around based on unrelated settings so that's why I suggested picking the one that works for most use cases (IMO, left). But from a code perspective, it shouldn't be that hard to make this dynamic.

@allylane Could you please verify if there are more scenarios we haven't covered that makes the policy links or copyright look off?

Are we okay with always left or do we want to introduce a setting-dependant dynamic solution?

**Review by sofiamatulis — APPROVED** [review]
Works as expected 🚢

File: assets/section-footer.css
Comment (LucasLacerdaUX) [reply assets/section-footer.css:497]: @ludoboludo I've updated to center-align whenever both language/currency settings are off. This should also account for when it's empty.

**Review by sofiamatulis — APPROVED** [review]

File: assets/section-footer.css
Comment (ludoboludo) [inline assets/section-footer.css:380]: I think this needs to be `.footer__copyright:not(.footer__content-bottom-wrapper--center)` no ? 
Otherwise it always overwrite the text alignment on screens wider than 750px. 

There is a chance for this to happen: 
<details>
<summary>Screenshot</summary>


</details>

**Pull request merged**
